artemis-commons/src/test/java/org/apache/activemq/artemis/utils/ByteUtilTest.java

```diff
@@ -376,6 +367,21 @@ public void testIntToByte() {
       }
    }
 
+   @Test
+   public void testLongToByte() {
+      for (int i = 0; i < 1000; i++) {
```

Comment:
I'm just copying it from another test. testIntToByte is the same here.

Reply:
I'm happy to suggest it would be better if updated too then ;)

Reply:
I have been bitten by predictable values in a test before.. I would prefer to have it random, even if that means eventual failures. probably at this point I'm just being superstitious :) lol

Reply:
I will change it though

Reply:
Preditable is good in the sense that you fully verify an expected case and you can see entirely what that case is, and very likely any failure will make it immediately clear what went wrong. As-is, if it fails youd have to first work back to try and even figure out what the long even was and then which bit was wrong etc. Ok, its not likely to be ByteBuffer, but its still using random other code to generate a value and verify whats already a known expected outcome.

If you want some random you could always do both....have specific [corner] case(s) with known byte result(s) to fully verify their expected behaviour, and a random for some randomness.

Mainly I was just saying it doesnt seem like it needs 1000 attempts on every test run rather than a few. I dont think thats likely to provide any more confidence in it really. I mean, ok its going to be silly-fast, but its still not really all that useful.